integrations/GoogleOptimize/util.js

```diff
@@ -0,0 +1,13 @@
+const ScriptLoaderHead = (id, src, async = defaultAsyncState) => {
```

Comment:
Let's update the common script loader utility

Reply:
updated